code/game/area/areas.dm

```diff
@@ -23,6 +23,7 @@
 									'sound/ambience/ambigen8.ogg','sound/ambience/ambigen9.ogg',\
 									'sound/ambience/ambigen10.ogg','sound/ambience/ambigen11.ogg',\
 									'sound/ambience/ambigen12.ogg','sound/ambience/ambigen14.ogg')
+	var/can_be_dirty = TRUE //Set this to FALSE to prevent an area from being made dirty at all
```

Comment:
#define CAN_BE_DIRTY 64 (rather than 64, a power of 2, which is not already being used by turf flags)

flags |= CAN_BE_DIRTY

then for logic checking, instead of (if(can_be_dirty)) is becomes (if(flags & CAN_BE_DIRTY))

Also make sure than phoebe lotsu has this flag.